homeassistant/components/subaru/const.py

```diff
@@ -32,9 +34,25 @@
 MANUFACTURER = "Subaru Corp."
 
 PLATFORMS = [
+    Platform.LOCK,
     Platform.SENSOR,
 ]
 
+SERVICE_LOCK = "lock"
+SERVICE_UNLOCK = "unlock"
+SERVICE_UNLOCK_SPECIFIC_DOOR = "unlock_specific_door"
+
+ATTR_DOOR = "door"
+
+UNLOCK_DOOR_ALL = "all"
+UNLOCK_DOOR_DRIVERS = "driver"
+UNLOCK_DOOR_TAILGATE = "tailgate"
+UNLOCK_VALID_DOORS = {
```

Comment:
It's easier to read if the constants that are only used in one module are defined in that module.